components/game/Game.tsx

```diff
@@ -30,13 +35,55 @@ function Game() {
         frameWidth: 120,
         frameHeight: 51,
       });
+      this.load.image("coin", "/coin.svg");
+      this.load.image("jewel", "/jewel.svg");
+      this.load.image("twistedBracelet", "/twistedBracelet.svg");
+      this.load.image("jewelBracelet", "/jewelBracelet.svg");
+      this.load.image("necklace", "/necklace.svg");
+      this.load.image("ruby", "/ruby.svg");
+      this.load.image("emerald", "/emerald.svg");
+      this.load.image("chalice", "/chalice.svg");
```

Comment:
You can reduce this if you make use of an array:
```suggestion
      // Array is not complete
      ['coin', 'jewel', 'twistedBracelet', 'jewelBracelet'].forEach((imageName) => {
            this.load.image(imageName, `/${imageName}.svg`);
      });
```